Content.Shared/DoAfter/Events.cs

```diff
@@ -0,0 +1,230 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.DoAfter;
+
+// This file contains a whole bunch DoAFterEvents specific to some server-side system / interaction.
+// Really these should all go into their respective shared namespaces whenever those systems get properly predicted.
```

Comment:
I'll probably fix this before marking the draft as ready, but its just such tedious work.